CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ This file ONLY contains changes made by the AIM17 dev-team. This means, that thi
 of the forked repository.
 The format is partially based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+## 2018-04-24 ([#39](https://github.com/amarbajric/EBUSA-AIM17/pull/39))
```

Comment:
Please change the url `https://github.com/amarbajric/EBUSA-AIM17/pull/39` to `https://github.com/amarbajric/EBUSA-AIM17/pull/68` as you are referring to your Pull Request :)
Also change the `#39` to `#68`.

So it should look like this:
`## 2018-04-24 ([#68](https://github.com/amarbajric/EBUSA-AIM17/pull/68))`